src/chrome/content/rules/Motorola.xml

```diff
@@ -63,23 +63,39 @@
 -->
 <ruleset name="Motorola (partial)">
 
-	<target host="*.email-motorola.com" />
 	<target host="*.motorola.com" />
 		<exclusion pattern="^http://(?:www\.)?motorola\.com/(?:Business|staticfiles)/" />
 	<target host="cdn.motorola-2.com" />
 	<target host="motorolasolutions.com" />
 	<target host="*.motorolasolutions.com" />
+	<target host="www.motorola.de" />
 
 
 	<securecookie host="^(?:globalrepairtools|mediacenter|membership)\.motorola\.com$" name=".+" />
 	<securecookie host="^mediacenter\.motorolasolutions\.com$" name=".+" />
-
-
-	<rule from="^http://[ls]\.email-motorola\.com/"
-		to="https://s.email-motorola.com/" />
-
+	
+	<test url="http://motorola.com/" />
+	<test url="http://www.motorola.com/" />
+
+	<test url="http://membership.motorola.com/" />
+	<test url="http://globalrepairtools.motorola.com/" />
+	<test url="http://mediacenter.motorola.com/" />
+	<test url="http://sdc.motorola.com/" />
+	
+	<test url="http://mediacenter.motorolasolutions.com/" />
+	
+	<test url="http://www.motorola.com/Business/" />
+	<test url="http://www.motorola.com/staticfiles/" />
+	<test url="http://motorola.com/staticfiles/" />
+	
+	<rule from="^http://motorolasolutions\.com/"
+		to="https://www.motorolasolutions.com/" />
+	
 	<rule from="^http://motorola\.com/"
 		to="https://www.motorola.com/" />
+		
+	<rule from="^http://www\.motorola\.de/"
+		to="https://www.motorola.de/" />
 
 	<rule from="^http://(globalrepairtools|mediacenter|membership|sdc|www)\.motorola\.com/"
 		to="https://$1.motorola.com/" />
```

Comment:
This doesn't seem to be possible.
After your recommended change 'test.sh' complains:

ERROR src/chrome/content/rules/Motorola.xml: Not enough tests (0 vs 1) for <Rule from '^http://cdn\.motorola-2\.com/' to 'https://cdn.motorola-2.com/'>
ERROR src/chrome/content/rules/Motorola.xml: Not enough tests (0 vs 1) for <Rule from '^http://mediacenter\.motorolasolutions\.com/' to 'https://mediacenter.motorolasolutions.com/'>